posthog/models/activity_logging/activity_log.py

```diff
@@ -77,6 +77,9 @@ class Meta:
     team_id = models.PositiveIntegerField(null=True)
     organization_id = models.UUIDField(null=True)
     user = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL)
+    # If truthy, user can be unset and this indicates a 'system' user made activity asynchronously
+    is_system = models.BooleanField(null=True)
```

Comment:
Do we need this? Don't we already have a user/person we're using as a system user for stuff like this?

Reply:
>  Don't we already have a user/person we're using as a system user for stuff like this?

Nope